server/src/main/resources/routes

```diff
@@ -106,6 +106,7 @@ DELETE        /staff-movements/:uuid
 
 # Feed import
 POST          /data/feed/red-list-counts                                              controllers.Application.feedImportRedListCounts()
++ nocsrf
```

Comment:
Just one + should work ?

Reply:
The first one is just github showing you it's a new line :)